### PR TITLE
Fix Typo and Draw circle only around planets visible in the viewport.

### PR DESCRIPTION
**Pull request by cristobal**
* Fix typo
* Only draw circle around those planets visible in the viewport.

### CODE REVIEW

**Review by modukon — COMMENTED** [review]
looks good, did you notice a performance improvement by only drawing the planets in viewport? will it also highlight planets that are in your view but too small to see?

**Comment by cristobal** [comment]
> looks good, did you notice a performance improvement by only drawing the planets in viewport? will it also highlight planets that are in your view but too small to see?

1. Yes i noticed a small improvement, still i guess for this round where we can capture fewer planets the improvement will not be that large. But for games where the gameplay allows for larger capture of planets it will improve.
2. Yes this will still highlight planets that are in the view but to small to see 👍🏽

**Comment by cristobal** [comment]
I also modfied the plugin for my taste not sure everyone agree but i prefer without fill, when using the `Highlight = Circe` e.g.:
Screenshot 2022-02-22 at 11 40 54

I can make another PR with a checkbox that allows for controlling fill or not, what do you think?

**Review by modukon — APPROVED** [review]

**Comment by cristobal** [comment]
Who can merge @modukon ?

File: content/casual/highlight-my-planets/plugin.js [hunk 1]
```diff
@@ -154,7 +154,9 @@ class Plugin {
     const origFillStyle = ctx.fillStyle;
     const origSrokeStyle = ctx.strokeStyle;
     const viewport = ui.getViewport();
-    const planets = df.getMyPlanets();
+    const planets = df.getMyPlanets().filter(
+      planet => viewport.isInViewport(planet.location.coords)
+    )
```
Comment (ichub) [inline content/casual/highlight-my-planets/plugin.js:159]: FYI, `GameUIManager#getPlanetsInViewport()` also exists, but this works.

**Review by ichub — APPROVED** [review]

**Pull request merged**